.commitlintrc.cjs

```diff
@@ -1,16 +1,26 @@
+const { execSync } = require('child_process')
```

Comment:
```suggestion
const { execSync } = require('child_process');
```

Reply:
Oh double quotes might need to be changed as well

Reply:
done ~
Damn I miss it